tools/downloader/src/open_model_zoo/model_tools/converter.py

```diff
@@ -104,11 +109,30 @@ def main():
 
     args = parser.parse_args()
 
+    try:
+        telemetry = tm.Telemetry('Model Converter', tid='UA-194864834-1')
```

Comment:
The tid is repeated everywhere; it should be moved to `common`.

Reply:
Moved to `common` as a `start_telemetry` parameter, although this tid is temporary, when everything is done and tested, this will be removed, since telemetry has tid of common base [backend_ga.py#L18](https://github.com/openvinotoolkit/telemetry/blob/master/src/backend/backend_ga.py#L18)